sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -164,10 +167,10 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
         auditUserId,
         application.id,
       );
-      let createdRestriction: StudentRestriction | undefined = undefined;
+      let createdRestrictions: StudentRestriction[] | undefined = undefined;
       if (studentRestriction) {
```

Comment:
Please rename `studentRestriction`  to `studentRestrictions` since it is an array now.